### DISCUSSION TITLE
Can Pulse view iPhone logs prior to its' installation?

### DISCUSSION THREAD

**Discussion by citycritter** [kean/Pulse#224] [Q&A] [answered]
Can Pulse view iPhone logs prior to its' installation?

**Comment by kean** [comment]
Can you please elaborate? Do you mean the logs recorded using OSLog or something else?

**Reply by citycritter** [reply]
If someone used a RAT tool on my iPhone within the past 2-3 months, could Pulse trace anything? Or what where to look?

**Reply by kean** [reply, marked as answer]
It shows only what's recorded in the Pulse database.
